cli/templatecreate.go

```diff
@@ -134,7 +134,7 @@ func (r *RootCmd) templateCreate() *clibase.Cmd {
 				VersionID:                  job.ID,
 				DefaultTTLMillis:           ptr.Ref(defaultTTL.Milliseconds()),
 				FailureTTLMillis:           ptr.Ref(failureTTL.Milliseconds()),
-				InactivityTTLMillis:        ptr.Ref(inactivityTTL.Milliseconds()),
+				TimeTilDormantMillis:       ptr.Ref(inactivityTTL.Milliseconds()),
```

Comment:
```suggestion
				TimeTilDormantMillis:       ptr.Ref(timeTilDormant.Milliseconds()),
```
I am not sure, but should this be renamed too?

Reply:
Probably but it's just a local variable and they mean the same thing.